core/src/test/scala/integration/kafka/api/PlaintextConsumerCommitTest.scala

```diff
@@ -304,6 +304,60 @@ class PlaintextConsumerCommitTest extends AbstractConsumerTest {
     consumeAndVerifyRecords(consumer = otherConsumer, numRecords = 1, startingOffset = 5, startingTimestamp = startingTimestamp)
   }
 
+  @ParameterizedTest(name = TestInfoUtils.TestWithParameterizedQuorumAndGroupProtocolNames)
+  @MethodSource(Array("getTestQuorumAndGroupProtocolParametersAll"))
+  def testEndOffsets(quorum: String, groupProtocol: String): Unit = {
```

Comment:
oh this one (and the one below) are related to partition's offsets, not committed offsets, so I would say they need to stay in the PlaintextConsumer, where you had them (I was only suggesting to move the `testSubscribeAndCommitSync` here, because it relates to committed offsets)

Reply:
Sorry - wasn't looking carefully at it.  Putting things back to the original place.